### PR TITLE
Add sign out component

### PR DESCRIPTION
**Pull request by lsh9034**
I add sign out control component!!

### CODE REVIEW

File: web/src/pages/signout.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+class SignOut extends Component {
+    render() {
+        Cookies.remove('token');
+        return (
+            <div>
+                logging out
+                {this.props.history.push('/')}
+            </div>
+            // <Redirect to='/'/>
```
Comment (mskwon1) [inline web/src/pages/signout.js]: delete this!

File: web/src/pages/signout.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        )
+    }
```
Comment (mskwon1) [inline web/src/pages/signout.js]: Indentation?

File: web/src/App.js [hunk 3]
```diff
@@ -13,20 +13,21 @@ import NavBar from './components/navigation/navbar';
 import React from 'react';
 import SignInUp from './pages/signInUp';
 import UserPage from './pages/userpage';
+import SignOut from './pages/signout';
```
Comment (shberry) [inline web/src/App.js]: Move this above UserPage

File: web/src/pages/signout.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+import React, {Component} from 'react';
+import {withRouter} from 'react-router-dom';
+
+class SignOut extends Component {
+  render() {
+    Cookies.remove('token');
+    return (
+      <div>
+          logging out
+        {this.props.history.push('/')}
```
Comment (shberry) [inline web/src/pages/signout.js]: Indentation looks off

**Pull request merged**